compiler/rustc_middle/src/mir/mono.rs

```diff
@@ -269,6 +270,14 @@ impl<'tcx> CodegenUnit<'tcx> {
         self.name = name;
     }
 
+    pub fn primary(&self) -> bool {
+        self.is_primary
+    }
+
+    pub fn set_primary(&mut self, primary: bool) {
```

Comment:
`set_primary` is only ever called with `true`, so perhaps specializing it to

```rust
pub fn make_primary(&mut self) { self.is_primary = true; }
```

would make sense?

Reply:
Indeed, thanks!